.github/labeler.yml

```diff
@@ -0,0 +1,223 @@
+documentation:
+  - docs/**/*
+  - any: [ '*.rst', '!CHANGES.rst' ]
+  - '*.md'
+  - .readthedocs.yml
+  - LICENSE
+
+installation:
+  - pyproject.toml
+  - 'setup.*'
+  - 'requirements-*.txt'
+  - MANIFEST.in
+
+ami:
+  - 'jwst/ami/**'
+
+assign_mtwcs:
+  - 'jwst/assign_mtwcs/**'
+
+assign_wcs:
+  - 'jwst/assign_wcs/**'
+
+associations:
+  - 'jwst/associations/**'
+
+background:
+  - 'jwst/background/**'
+
+barshadow:
+  - 'jwst/barshadow/**'
+
+combine_1d:
+  - 'jwst/combine_1d/**'
+
+coron3 pipeline:
```

Comment:
could add similar entries for the other pipelines: image2, image3, spec2, spec3, ami3, tso3, detector1